stable/metricbeat/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Helm chart to collect Kubernetes logs with metricbeat
 icon: https://www.elastic.co/assets/blt47799dcdcf08438d/logo-elastic-beats-lt.svg
 name: metricbeat
-version: 0.4.0
+version: 0.4.1
```

Comment:
please confirm the current chart version and bump it.

Reply:
Bumped!